### PR TITLE
Convert familiar abilities to the action item type

### PR DESCRIPTION
**Pull request by CarlosFdez**
Converts the familiar abilities in the compendium to the action item type, with the category set to familiar abilities. As per discussion, there is no identifier for master abilities. Also implements compendium searching for familiar abilities, and the condition/effects section hides when there are no conditions or effects on the familiar.

openActionTab() has a breaky change, but that's ok because 5.0 is around the corner. Ignore white space when looking at the familiar sheet styling diff.

image

image

### CODE REVIEW

File: packs/data/familiar-abilities.db/spell-delivery.json [hunk 1]
```diff
@@ -3,34 +3,33 @@
...
         },
-        "duration": {
-            "expiry": "turn-start",
-            "sustained": false,
-            "unit": "unlimited",
-            "value": -1
-        },
         "level": {
             "value": 1
         },
```
Comment (stwlam) [inline packs/data/familiar-abilities.db/spell-delivery.json:18]: ?

Comment (CarlosFdez) [reply packs/data/familiar-abilities.db/spell-delivery.json:18]: I think I assumed it was unused actor data, I shall remove.

File: static/templates/actors/familiar-sheet.hbs [hunk 2]
```diff
@@ -194,30 +194,63 @@
...
+                            <li class="item action flexrow" data-item-id="{{action.id}}">
+                                <div class="action-header">
+                                    <div class="action-name rollable">
+                                        <h4><a>{{action.name}} {{{actionGlyph action.actionCost}}}</a></h4>
+                                    </div>
+                                    <div class="item-controls">
+                                        <a class="item-chat chat" title="{{localize "PF2E.NPC.SendToChat"}}"><i class="fas fa-fw fa-comment-alt"></i></a>
+                                        {{#if @root.options.editable}}
+                                            <a class="item-control item-edit" title="{{localize "PF2E.EditItemTitle"}}"><i class="fas fa-fw fa-edit"></i></a>
+                                            <a class="item-control item-delete" title="{{localize "PF2E.DeleteItemTitle"}}"><i class="fas fa-fw fa-trash"></i></a>
```
Comment (stwlam) [inline static/templates/actors/familiar-sheet.hbs:218]:
```suggestion
                                        <a class="item-chat chat" title="{{localize "PF2E.NPC.SendToChat"}}"><i class="fa-solid fa-fw fa-comment-alt"></i></a>
                                        {{#if @root.options.editable}}
                                            <a class="item-control item-edit" title="{{localize "PF2E.EditItemTitle"}}"><i class="fa-solid fa-fw fa-edit"></i></a>
                                            <a class="item-control item-delete" title="{{localize "PF2E.DeleteItemTitle"}}"><i class="fa-solid fa-fw fa-trash"></i></a>
```

File: static/templates/actors/familiar-sheet.hbs [hunk 3]
```diff
@@ -194,30 +194,63 @@
...
+
             <!-- ABILITIES -->
-            <!-- coming soon... -->
-            <!-- CONDITIONS -->
             <div class="section-container">
                 <div class="section-header">
-                    <h4>{{localize "PF2E.ConditionsLabel"}}</h4>
+                    <h4>{{localize "PF2E.Familiar.AbilitiesLabel"}}</h4>
+                    <div class="controls">
+                        <a class="action-browse" data-action-category="familiar"><i class="fa-solid fa-plus"></i> {{localize "PF2E.BrowseLabel"}}</a>
```
Comment (stwlam) [inline static/templates/actors/familiar-sheet.hbs:203]:
```suggestion
                        <a class="action-browse" data-action-category="familiar"><i class="fa-solid fa-search"></i> {{localize "PF2E.BrowseLabel"}}</a>
```

**Pull request merged**